lib/fog/ovirt/models/compute/cluster.rb

```diff
@@ -5,8 +5,59 @@ class Cluster < Fog::Model
         identity :id
 
         attribute :name
+        attribute :version
         attribute :raw
 
+        # Max resources values for VMs depending on oVirt's version
+        # These values can be found by running the engine-config command on the engine
+        # with following config keys:
+        # - MaxNumOfVmCpus
+        # - MaxNumOfVmSockets
+        # - VM64BitMaxMemorySizeInMB
+        # - VM32BitMaxMemorySizeInMB,
+        #
+        # Eg:
+        #
+        # engine-config --get MaxNumOfVmCpus
+        # MaxNumOfVmCpus: 288 version: 4.1
+        # MaxNumOfVmCpus: 384 version: 4.2
+        # MaxNumOfVmCpus: 384 version: 4.3
+        #
+        VM_MAX_VALUES = {
+          "4.1" => { :max_num_of_vm_cpus => 288,
+                     :max_num_of_vm_sockets => 16,
+                     :vm64_bit_max_memory_size_in_mb => 4_194_304,
+                     :vm32_bit_max_memory_size_in_mb => 2048 },
```

Comment:
Shouldn't this (and all three others) be 20_480?

Reply:
You are right, sorry.